api/types/registry/registry.go

```diff
@@ -92,8 +92,10 @@ type SearchResult struct {
 	IsOfficial bool `json:"is_official"`
 	// Name is the name of the repository
 	Name string `json:"name"`
-	// IsAutomated indicates whether the result is automated
-	IsAutomated bool `json:"is_automated"`
+	// IsAutomated indicates whether the result is automated.
+	//
+	// Deprecated: the "is_automated" field is being deprecated by Docker Hub's search API, and will no longer be set in future.
+	IsAutomated bool `json:"is_automated,omitempty"`
```

Comment:
Making the field `omitempty` is a breaking change for some clients which don't use Go's JSON parser. We should be nice to third-party clients by only omitting the field in responses to requests with the latest API version, leaving `"is_automated": false` in responses for older API versions.

Reply:
Ah, you're right yes.

Reply:
updated; PTAL